src/main/java/co/cask/wrangler/Wrangler.java

```diff
@@ -59,6 +60,9 @@
   // Output Schema associated with transform output.
   private Schema oSchema;
 
+  // Error counter.
+  private long errorCounter;
```

Comment:
if I make it static them it needs to be AtomicLong -- do we need to make that. Remember multiple instance could be running in the same JVM.